### PR TITLE
Fix responsive et autres features

### PR DESCRIPTION
**Pull request by seinhorn**
- responsive design global (principalement en mode mobile landscape)
- ajouts graphiques : posts, author, pagination
- ajout de traduction
...

### CODE REVIEW

**Comment by ElevenWilson** [comment]
6 words might be misspelled, please check them: blockquote, breakpoint, li, pre, px, ul

File: _sass/objects/_meta.scss [hunk 1]
```diff
@@ -1,39 +1,74 @@
...
+    display: inline-block;
+
+    .page-heading & {
+      margin-bottom: 1rem;
+    }
+
+    &::after {
+      content: '';
+      font-size: 1.4rem;
+      margin: 0 .6rem;
```
Comment (ElevenTom) [inline _sass/objects/_meta.scss]: rajouter  display: inline-block;

File: _sass/objects/_meta.scss [hunk 2]
```diff
@@ -1,39 +1,74 @@
...
+
+  &-item {
+    display: inline-block;
+
+    .page-heading & {
+      margin-bottom: 1rem;
+    }
+
+    &::after {
+      content: '';
```
Comment (ElevenTom) [inline _sass/objects/_meta.scss]: rajouter après content " display: inline-block; "

**Comment by ElevenWilson** [comment]
6 words might be misspelled, please check them: blockquote, breakpoint, li, pre, px, ul

**Comment by ElevenWilson** [comment]
6 words might be misspelled, please check them: blockquote, breakpoint, li, pre, px, ul

**Pull request merged**